components/count-badge/README.md

```diff
@@ -46,7 +46,7 @@ The `d2l-count-badge` element is a web component to display a number count, depe
 | `number` | Number, required | The number to display on the badge.  Must be a positive integer. |
 | `size`, default: `small` | String | The size of the badge. Valid options are `"small"` and `"large"`. |
 | `type`, default: `count` | String | The type of the badge. Valid options are `"notification"` and `"count"`. Notification badges are orange and count badges are grey. |
-| `max-digits`, default: `2` when `type="notification"` | Number | Optionally specify a digit limit, after which numbers are truncated. Defaults to two for `"notification"` type and no limit for `"count"` type.
+| `max-digits`, default: `2` when `type="notification"`, `5` when `type="count"` | Number | Optionally specify a digit limit, after which numbers are truncated. Defaults to two for `"notification"` type and five for `"count"` type. Must be between 1-5.
```

Comment:
Is there still a reason to even expose this as an attribute? Like are there valid examples where folks want `4`, or are our defaults of 2 and 5 now good enough? Just trying to keep the components as simple as possible.

Reply:
There are cases, like filter, where the consumers want the count-type styling and a 2 digit limit. I think portfolio does the same, so I think we do need to keep it exposed.

Reply:
Gotcha -- thanks!